import-recipes/recipes.json

```diff
@@ -12,16 +12,6 @@
             "id": "city",
             "name": "City"
           },
-          {
-            "href": "",
-            "id": "country",
-            "name": "Country"
-          },
-          {
-            "href": "",
-            "id": "siblings",
-            "name": "Number Of Siblings"
-          },
```

Comment:
Just confirming it was intentionally this was removed?

Reply:
Yes this was intentional